project/DependsOn.scala

```diff
@@ -0,0 +1,29 @@
+import sbt._
+
+object DependsOn {
+
+  def scalatest(scalaVersion: String): ModuleID = {
+    val artifact = "org.scalatest" %% "scalatest"
+    if (scalaVersion.startsWith("2.12")) artifact % "3.0.0" % Test
```

Comment:
@pathikrit this is an sbt build script, do you really want a constant for 2.12 ?